js/src/get-started-page/get-started-card/index.scss

```diff
@@ -1,41 +1,60 @@
-.woocommerce-marketing-google-get-started-card {
-	.components-flex {
-		flex-direction: column;
-		align-items: flex-end;
+@import "../_shared";
 
-		@include break-large() {
-			flex-direction: row;
-			align-items: center;
-		}
+.gla-get-started-card {
+	display: flex;
+	flex-direction: row-reverse;
+	padding: $grid-unit-50 $grid-unit-50 $grid-unit-20 $grid-unit-50;
 
-		.motivation-text {
-			margin: calc(var(--main-gap) * 2.5);
+	@media (max-width: $break-small) {
+		flex-direction: column;
+		padding: $grid-unit-30;
+	}
 
-			.title {
-				margin-bottom: $grid-unit-20;
-			}
+	.motivation-image {
+		min-width: auto;
 
-			.description {
-				margin-bottom: $grid-unit-20;
-			}
+		@media (max-width: $break-small) {
+			text-align: center;
 		}
+	}
 
-		.motivation-image {
-			text-align: right;
-			margin-bottom: -4px;
+	// Adjust <CardBody> imported from @wordpress/components.
+	// Repeat selector to make it higher priority.
+	.components-card__body.components-card__body {
+		display: flex;
+		flex-direction: column;
+		align-items: flex-start;
+		gap: 20px;
+		padding: 0;
 
-			@include break-large() {
-				margin-top: calc(var(--main-gap) * 1.5);
-			}
+		@media (max-width: $break-small) {
+			gap: 16px;
+			text-align: center;
+			align-items: center;
 		}
 	}
 
-	.components-flex &__terms-notice {
-		margin-top: $grid-unit-20;
-		font-style: italic;
+	// Use more specific rules to make it higher priority to override <Text> component.
+	& &__terms-notice {
+		// margin-top: $grid-unit-20;
```

Comment:
💅  The commented-out style seems to be left?

Reply:
Ooops, updated in a819c2e, thanks!